tests/xmap_test.py

```diff
@@ -115,96 +90,14 @@ def new_f(*args, **kwargs):
   return new_f
 
 
-class GmapTest(jtu.JaxTestCase):
-
-  @check_default_schedules(lambda _: True)
-  @skip_insufficient_devices(8)
-  @ignore_gmap_warning()
-  @skipIf(not config.omnistaging_enabled,
-          "vmap collectives only supported when omnistaging is enabled")
-  def testBasicSchedules(self, schedule):
-    def f(x):
-      return jnp.dot(jnp.sin(x), x.T) * 4 + x
-
-    x = jnp.arange(800).reshape((8, 10, 10))
-
-    self.assertAllClose(gmap(f, schedule)(x), vmap(f)(x))
-
-  @check_default_schedules(lambda s: not any(c[0] == 'sequential' for c in s))
-  @skip_insufficient_devices(8)
-  @ignore_gmap_warning()
-  @skipIf(not config.omnistaging_enabled,
-          "vmap collectives only supported when omnistaging is enabled")
-  def testAxisName(self, schedule):
-    def f(x):
-      return x - lax.psum(x, 'i')
-    x = jnp.arange(8)
-    self.assertAllClose(gmap(f, schedule, axis_name='i')(x),
-                        vmap(f, axis_name='i')(x))
-
-  @ignore_gmap_warning()
-  @skipIf(not config.omnistaging_enabled,
-          "vmap collectives only supported when omnistaging is enabled")
-  def testAxisName2d(self):
-    def f(x):
-      return x - lax.psum(x, 'i') + lax.pmax(x, 'j')
-    x = jnp.arange(8 * 8).reshape((8, 8))
-    s = [('vectorized', None)]
-    self.assertAllClose(gmap(gmap(f, s, axis_name='i'), s, axis_name='j')(x),
-                        vmap(vmap(f, axis_name='i'), axis_name='j')(x))
-
 class XMapTest(jtu.JaxTestCase):
 
-  @ignore_gmap_warning()
-  @skipIf(not config.omnistaging_enabled,
-          "vmap collectives only supported when omnistaging is enabled")
-  @with_mesh([('r1', 4), ('r2', 2), ('r3', 5)])
-  def testXMap(self):
-    def f(a, b):
-      return a + 2, b * 4
-    fm = xmap(f,
-              in_axes=[A({'x': 0, 'z': 1}), A({'y': 1})],
-              out_axes=[A({'x': 1, 'z': 0}), A({'y': 0})],
-              schedule=[
-                ('x', 'r1'),
-                ('x', 'r2'),
-                ('y', 'r1'),
-                ('z', 'r3'),
-                ('x', 'vectorize'),
-                ('y', 'vectorize'),
-              ])
-    a = jnp.arange(16*5*2).reshape((16, 5, 2))
-    b = jnp.arange(6*16).reshape((6, 16))
-    c, d = fm(a, b)
-    self.assertAllClose(c, (a + 2).transpose((1, 0, 2)))
-    self.assertAllClose(d, (b * 4).T)
-
-  @ignore_gmap_warning()
-  @skipIf(not config.omnistaging_enabled,
-          "vmap collectives only supported when omnistaging is enabled")
-  @with_mesh([('r1', 4), ('r2', 2), ('r3', 5)])
-  def testXMapCollectives(self):
```

Comment:
Note that those two tests were always skipped anyway, because none of our test environments has 40 devices available